tests/src/test/java/alluxio/worker/AlluxioWorkerRestApiTest.java

```diff
@@ -91,6 +97,37 @@ public void getCapacityBytesTest() throws Exception {
         NO_PARAMS, HttpMethod.GET, capacityBytes).run();
   }
 
+  private Configuration mockConfiguration() {
```

Comment:
Powermock adds a lot of complexity and is more error-prone than testing without Powermock. We generally try to limit its usage to times when it is the only option.